addons/website/static/src/js/menu/edit.js

```diff
@@ -72,9 +72,18 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
     start: function () {
         var def = this._super.apply(this, arguments);
 
+        let navbar = $('nav#oe_main_menu_navbar').get(0);
+        let stopPropagation = function (event) {
+            event.stopPropagation();
+        };
+
         // If we auto start the editor, do not show a welcome message
         if (this._editorAutoStart) {
-            return Promise.all([def, this._startEditMode()]);
+            navbar.addEventListener('click', stopPropagation, {capture: true});
```

Comment:
```suggestion
            navbar.addEventListener('click', stopPropagation, true);
```

Reply:
Ok, I like the options form better though.

Reply:
You can keep it like that it is a minor remark :+1: , don't really mind it..
I just checked on our codebase how we were writing it: with the direct form. Plus saw the doc that suggest a direct form when only setting capture to true